README.md

```diff
@@ -38,70 +38,95 @@ The second: you need not only a secure context for the browser, but actual safet
 
 ## Overview
 
-1. Get a server. It doesn't need to be big. Ideally you should have at least one slave, too, because NS entries require at least two servers.
-1. Point the NS entry of your domain to this server.
-1. [Install deps](#base-installation-and-deps).
-1. [Run dnsserver.py](#running-the-dns-server).
-1. [Create the certificates running certbotdns.py](#renewing-keys).
+1. Get an Ubuntu server. It doesn't need to be big. Ideally you should have at least one slave, too, because NS entries require at least two servers.
+2. Point the NS entry of your domain to this server.
+3. [Install deps](#base-installation-and-deps).
+4. [Run dnsserver.py](#running-the-dns-server).
+5. [Create the certificates running certbotdns.py](#renewing-keys).
+
+## Prerequisites 
+
+* Python 3.6 or above (see `util/ubuntu-install.bash` script )
+* certbot and the dnslib and cherrypy PIPs (see `util/ubuntu-install.bash` script )
+* Ubuntu 22.04 Server
```

Comment:
same here. Nothing against a `ubuntu-install` script or detailed instructions for it, but if possible let's keep docs as general as possible.

Reply:
ah - good call!  I've added back the old one and called it `ubuntu1804-install.bash` and renamed the new one to `ubuntu2204-install.bash`.  See a5beab0 and 052a11c